torch/_subclasses/fake_tensor.py

```diff
@@ -1124,8 +1124,12 @@ def merge_devices(t):
                 f"Unhandled FakeTensor Device Propagation for {func}, found two different devices {common_device}, {t.device}"
             )
 
-        tree_map(merge_devices, args)
-        tree_map(merge_devices, kwargs)
+        if func is aten.copy_.default and args and isinstance(args[0], FakeTensor):
```

Comment:
register it here instead ? https://github.com/pytorch/pytorch/blob/2319e5f64075cf318b1edda839eac0723b2a10be/torch/_subclasses/fake_tensor.py#L455